rust/psibase/src/services/evaluations.rs

```diff
@@ -0,0 +1,141 @@
+#[crate::service(name = "eval-hooks", dispatch = false, psibase_mod = "crate")]
+#[allow(non_snake_case, unused_variables)]
+pub mod Hooks {
+    use crate::AccountNumber;
+
+    #[action]
+    fn evalRegister(evaluation_id: u32, account: AccountNumber) {
+        unimplemented!()
+    }
+
+    #[action]
+    fn evalUnregister(evaluation_id: u32, account: AccountNumber) {
+        unimplemented!()
+    }
+
+    #[action]
+    fn evalGroupFin(evaluation_id: u32, group_number: u32, result: Vec<u8>) {
+        unimplemented!()
+    }
+
+    /// Creates and schedules a new evaluation with specified phases and parameters.
+    ///
+    /// # Arguments
+    /// * `registration` - Unix seconds timestamp for the start of the registration phase.
+    /// * `deliberation` - Unix seconds timestamp for the start of the deliberation phase.
+    /// * `submission` - Unix seconds timestamp for the start of the submission phase.
+    /// * `finish_by` - Unix seconds timestamp for the start of the evaluation completion phase.
+    /// * `allowed_group_sizes` - Vector of allowed group sizes (sizes must be greater than 0, traditionally [4, 5, 6]).
+    /// * `num_options` - Number of options available for proposals (traditionally 6).
+    /// * `use_hooks` - Flag to enable or disable hooks for the evaluation.
+    ///
+    /// # Returns
+    /// The ID of the newly created evaluation.    
+    #[action]
+    fn create(
```

Comment:
You added all of these actions into a service called Hooks. Only the hooks should be in the Hooks module.

Reply:
I've attempted to sort this out, can you see my changes in https://github.com/gofractally/psibase/pull/1171 ? @James-Mart